add stack exchange (#2803)
huggingface/datasets: datasets/stack_exchange/README.md

---
annotations_creators:
- no-annotation
language_creators:
- found
languages:
- en
licenses:
- mit
multilinguality:
- monolingual
pretty_name: Stack Exchange
size_categories:

> - unknown

Reviewer: ```suggestion
- 1M<n<10M
```

    - [Data Instances](#data-instances)
    - [Data Fields](#data-fields)
    - [Data Splits](#data-splits)
  - [|split|num examples|](#splitnum-examples)
  - [Dataset Creation](#dataset-creation)
    - [Curation Rationale](#curation-rationale)
    - [Source Data](#source-data)
      - [Initial Data Collection and Normalization](#initial-data-collection-and-normalization)
      - [Who are the source language producers?](#who-are-the-source-language-producers)
    - [Annotations](#annotations)
      - [Annotation process](#annotation-process)
      - [Who are the annotators?](#who-are-the-annotators)
    - [Personal and Sensitive Information](#personal-and-sensitive-information)
  - [Considerations for Using the Data](#considerations-for-using-the-data)
    - [Social Impact of Dataset](#social-impact-of-dataset)
    - [Discussion of Biases](#discussion-of-biases)
    - [Other Known Limitations](#other-known-limitations)
  - [Additional Information](#additional-information)
    - [Dataset Curators](#dataset-curators)
    - [Licensing Information](#licensing-information)
    - [Citation Information](#citation-information)
    - [Contributions](#contributions)

## Dataset Description

> - **Homepage:** https://github.com/EleutherAI/stackexchange-dataset
> - **Repository:** [Needs More Information]
> - **Paper:** https://arxiv.org/abs/2101.00027

Reviewer: ```suggestion
- **Homepage:** [GitHub](https://github.com/EleutherAI/stackexchange-dataset)
- **Repository:** [Needs More Information]
- **Paper:** [arXiv](https://arxiv.org/abs/2101.00027)
```

- **Leaderboard:** [Needs More Information]
- **Point of Contact:** [Needs More Information]

### Dataset Summary

This dataset is part of EleutherAI/The Pile dataset and is a dataset for Language Models from processing stackexchange data dump, which is an anonymized dump of all user-contributed content on the Stack Exchange network.

|download_size|34.28 Gib|
|dataset_size|10.3 Gib|

### Supported Tasks and Leaderboards

> - `lm`

Reviewer: ```suggestion
The dataset is used for Language Modeling.
```

### Languages

> - `en`

Reviewer: ```suggestion
The dataset is in English.
```

```

### Data Fields

> - `domain`
> - `text`

Reviewer: ```suggestion
- `domain`: Stack Exchange domain of the sample
- `text`: Text content containing both the question and the answer
```

## Dataset Creation

### Curation Rationale

[Needs More Information]

### Source Data

#### Initial Data Collection and Normalization

[Needs More Information]

#### Who are the source language producers?

[Needs More Information]

### Annotations

#### Annotation process

[Needs More Information]

#### Who are the annotators?

[Needs More Information]

### Personal and Sensitive Information

[Needs More Information]

## Considerations for Using the Data

### Social Impact of Dataset

[Needs More Information]

### Discussion of Biases

[Needs More Information]

### Other Known Limitations

[Needs More Information]

## Additional Information

### Dataset Curators

[Needs More Information]

### Licensing Information

[Needs More Information]

### Citation Information

```
@article{pile,
    title={The {P}ile: An 800GB Dataset of Diverse Text for Language Modeling},
    author={Gao, Leo and Biderman, Stella and Black, Sid and Golding, Laurence and Hoppe, Travis and Foster, Charles and Phang, Jason and He, Horace and Thite, Anish and Nabeshima, Noa and Presser, Shawn and Leahy, Connor},
    journal={arXiv preprint arXiv:2101.00027},
    year={2020}
}
```

### Contributions

> [sdtblck](https://github.com/sdtblck) creates the dataset, and [richarddwang](https://github.com/richarddwang) adds the dataset.

Reviewer: ```suggestion
Thanks to [sdtblck](https://github.com/sdtblck) for creating the dataset.
Thanks to [richarddwang](https://github.com/richarddwang) for adding the dataset.
```
